bin/bootstrap

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set -e
+
+cd "$(dirname "$0")/.."
+
+if [ -f "Brewfile" ] && [ "$(uname -s)" = "Darwin" ]; then
+  echo "==> Checking for Homebrew updates…"
+  brew update
+
+  brew bundle check >/dev/null 2>&1  || {
+    echo "==> Installing Homebrew dependencies…"
+    brew bundle
+
+    echo "==> Installing Ruby…"
+    brew bootstrap-rbenv-ruby
+
+    echo "==> Installing Node…"
+    brew bootstrap-nodenv-node
```

Comment:
Probably want to pull the `bootstrap-rbenv-ruby` and `bootstrap-nodenv-node` outside of the `brew bundle check`